docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -175,6 +175,11 @@ app.MapPost("/complete-sale", ([FromBody] SaleModel model, HatCoMetrics metrics)
 });
 ```
 
+### Best Practices
+
+- The <xref:System.Diagnostics.Metrics.Meter?displayProperty=nameWithType> does implement IDisposable, but IMeterFactory will manage the lifetime of any Meters it
```

Comment:
- <xref:System.Diagnostics.Metrics.Meter?displayProperty=nameWithType> implements IDisposable, but IMeterFactory automatically manages the lifetime of any Meters it
  creates, disposing them when the DI container is disposed. Adding extra code to invoke Dispose() on the Meter is unnecessary and will not have any effect.